index.html

```diff
@@ -52,6 +61,234 @@ <h2 class="sub-title">I am a Softwaer Developer</h2>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+            
+        </ul>
+    </div>
+    <div class="card-last-button">See Project</div>
+
+   </section>
+   <!--professional part start 3-->
+   <section class="card2">
+    <div class="card-title">Profesional Art<br> Printing Data</div>
+    <div class="card-pro-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+    <div class="card-pro-buttons">
+        <ul class="pro-buttons">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+            
+        </ul>
+    </div>
+    <div class="card-last-button">See Project</div>
+
+   </section>
+   <!--professional part start 4-->
+   <section class="card2">
+    <div class="card-title">Profesional Art<br> Printing Data</div>
+    <div class="card-pro-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+    <div class="card-pro-buttons">
+        <ul class="pro-buttons">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+            
+        </ul>
+    </div>
+    <div class="card-last-button">See Project</div>
+
+   </section>
+   <!--professional part start 5-->
+   <section class="card2">
+    <div class="card-title">Profesional Art<br> Printing Data</div>
+    <div class="card-pro-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+    <div class="card-pro-buttons">
+        <ul class="pro-buttons">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+            
+        </ul>
+    </div>
+    <div class="card-last-button">See Project</div>
+
+   </section>
+   <!--professional part start 6-->
+   <section class="card2">
+    <div class="card-title">Profesional Art<br> Printing Data</div>
+    <div class="card-pro-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</div>
+    <div class="card-pro-buttons">
+        <ul class="pro-buttons">
+            <li><a href="#">html</a></li>
+            <li><a href="#">css</a></li>
+            <li><a href="#">javascript</a></li>
+            
+        </ul>
+    </div>
+    <div class="card-last-button">See Project</div>
+
+   </section>
+   <!--about section start-->
+   <section class="card-3">
+    <div class="card-about-title">About me</div>
+    <div class="card-about-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
```

Comment:
- [ ] Please make sure that you use a `<p>` tag for your paragraph over `<div>`.  For more clarity, please visit above comments 👆